.github/workflows/ci.yml

```diff
@@ -19,8 +19,6 @@ jobs:
       with:
         path: ~/.nuget/packages
         key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
-        restore-keys: |
-          ${{ runner.os }}-nuget-
```

Comment:
Is this not necessary? I was under the understanding that without this it would not fallback to the "most generic" cache, expecting an exact match instead.

Not against it, just double checking.

Reply:
I'd rather treat it as a cache miss than fallback.  Caches will keep expanding (old stuff with new stuff) if there's a fallback.   Maybe worth a discussion